openpype/modules/kitsu/plugins/publish/validate_kitsu_intent.py

```diff
@@ -0,0 +1,27 @@
+# -*- coding: utf-8 -*-
+import pyblish.api
+import gazu
+
+
+class ValidateKitsuIntent(pyblish.api.ContextPlugin):
+    """Validate Kitsu Status"""
+
+    order = pyblish.api.ValidatorOrder
+    label = "Kitsu Intent/Status"
+    # families = ["kitsu"]
+    optional = True
+
+    def process(self, context):
+
+        publish_status = context.data.get("intent", {}).get("value")
+        if not publish_status:
+            self.log.info("Status is not set.")
+
```

Comment:
It should probably stop processing if intent is not available on context.

Reply:
Done.